xhttp/xhttpserver/tls_test.go

```diff
@@ -95,6 +506,9 @@ func testNewTlsConfigWithClientCACertificateFile(t *testing.T) {
 			CertificateFile:         "server.cert",
 			KeyFile:                 "server.key",
 			ClientCACertificateFile: "server.cert",
+			PeerVerify: PeerVerifyOptions{
+				CommonNames: []string{"Hippies, Inc."},
```

Comment:
☮️

Reply:
Did you intend to add a quick sanity check the PeerVerify is setup correctly here?

Reply:
That really isn't necessary, since the act of testing itself will verify that.  Also, the `PeerVerifyOptions` struct really doesn't have any rules around setting it up.  It's just a tuple of slices of strings.